cs_upload.py

```diff
@@ -430,7 +430,14 @@ def upload_to_s3(self, form_data: str, trace_file: str) -> None:
                     f"Unable to parse the response of the backend:\n{output}",
                 )
                 exit(1)
-            xml_key = xml.find("Key")
+            # SeaweedFS object storage response includes XML namespace, while MinIO
+            # does not. Use regular expression to check if `xmlns="<VALUE>"` is present
+            # in reponse string and capture it.
+            xmlns = ""
+            match = re.search(r'xmlns="(?P<value>.*?)"', output)
+            if match is not None:
+                xmlns = f"{{{match.groupdict()['value']}}}"
```

Comment:
**Suggestion (non-blocking):** Use `match.groups("value")`

Imho this is slightly more readable then using `groupdict`. YMMV, and since this is largely a matter of style, just keep it that way if you feel so.

Reply:
Followed your suggestion!